llvm/lib/Transforms/Vectorize/LoopVectorizationPlanner.h

```diff
@@ -365,8 +365,8 @@ class LoopVectorizationPlanner {
   /// Return the best VPlan for \p VF.
   VPlan &getBestPlanFor(ElementCount VF) const;
```

Comment:
```suggestion
  /// Return the VPlan for \p VF.
  VPlan &getPlanFor(ElementCount VF) const;
```
would be more accurate, independent of this patch.

Reply:
Will do separately, thanks!